visualization/app/codeCharta/ui/ribbonBar/artificialIntelligence/suspiciousMetrics/suspiciousMetricsMenu/suspiciousMetricsMenu.component.html

```diff
@@ -0,0 +1,20 @@
+<div (click)="$event.stopPropagation()">
+    <div>
+        <div class="title">Suspicious Metrics</div>
+        <div class="title title-button-container">
+            <i
+                class="fa fa-question-circle-o title-button"
+                role="button"
+                title="Open Information about Suspicious Metrics"
+                popovertarget="suspiciousMetricPopover"
+                (click)="openDialog()"
+            ></i>
```

Comment:
The hitbox for the above button is smaller than the icon suggests, only inside of the white circle the dialog gets opened, the blue area around it looks to be clickable but nothing happens. This is different from our other buttons that look similar